libfreerdp/core/smartcardlogon.c

```diff
@@ -280,10 +283,19 @@ static BOOL list_provider_keys(const rdpSettings* settings, NCRYPT_PROV_HANDLE p
 		                       NULL) <= 0)
 			goto endofloop;
 
+		WLog_DBG(TAG, "opening key %s", cert->info.containerName);
+
 		status = NCryptOpenKey(provider, &phKey, keyName->pszName, keyName->dwLegacyKeySpec,
-		                       keyName->dwFlags);
+		                       NCRYPT_SILENT_FLAG);
```

Comment:
Any reason to change `keyName->dwFlags` to `NCRYPT_SILENT_FLAG`?

Reply:
Yeah, the return flags of `NCryptEnumKeys` gives information, they aren't intented to be use later for `NCryptOpenKey`. I have  a windows system where it breaks the key listing, `NCRYPT_SILENT_FLAG` is enough.

Reply:
ok, so not by accident :+1: